client/src/components/APIDataTypes.ts

```diff
@@ -0,0 +1,138 @@
+// TYPES ASSOCIATED WITH INPUT DATA:
+
+export type Borough = "MANHATTAN" | "BRONX" | "BROOKLYN" | "QUEENS" | "STATEN ISLAND" | null;
```

Comment:
Ugh, it feels icky to have to include a `null` option here as the boro field should never really be null. That being said, the `makeEmptySearchAddress()` function in [AddressSearch.tsx](https://github.com/JustFixNYC/who-owns-what/blob/master/client/src/components/AddressSearch.tsx) involves creating an empty SearchAddress object, which is why I introduced the null option to the boro type. The other alternative I see would be to actually assign a borough within this `makeEmptySearchAddress()` function, but not sure whether that would confuse code readers and/or mess up some functionality elsewhere. Any thoughts?

Reply:
Ah ok, ill look into refactoring it closer to its use in the `makeEmptySearchAddress()` function. And ooh cool I didn't know you could add JSDoc-style comments to properties, I thought it was only for functions!